pages/national_resources/gr_resources.md

```diff
@@ -1,13 +1,13 @@
 ---
 
 training:
-  - name: ELIXIR GREECE Zenodo
+  - name: "ELIXIR GREECE: Prospects of data and tools for ELIXIR Greece"
     registry: Zenodo
-    description: "ELIXIR GREECE: Prospects of data and tools for ELIXIR Greece"
+    description: This presentation provides and overview of the data, tools and standards used in ELIXIR Greece as well as guidelines for effective research data management, emphasizing on best practices for open science and data stewardship.
```

Comment:
Here's a revised version with tiny corrections: 
This presentation provides an overview of the data, tools, and standards used in ELIXIR Greece, as well as guidelines for effective research data management, with an emphasis on best practices for open science and data stewardship